Library/Formula/ltsa.rb

```diff
@@ -0,0 +1,19 @@
+class Ltsa < Formula
+  desc "LTS Analyser"
+  homepage "http://www.doc.ic.ac.uk/~jnm/book/ltsa/LTSA_applet.html"
+  url "http://www.doc.ic.ac.uk/~jnm/book/ltsa/ltsatool.zip"
```

Comment:
Can I ask why this requirement exists?

I could understand it if the SHA sum wasn't required/given, but presumably it's not even (programmatically) checked to be equal to the supplied version - since the cookbook states that if auto-detection fails, the version must be supplied manually.